lib/types/cql-utils.js

```diff
@@ -7,9 +7,64 @@ const TimeUuid = require("./time-uuid");
 const Uuid = require("./uuid");
 const Duration = require("./duration");
 
+/**
+ * Ensures the value isn't one of many ways to express null value
+ * @param {string} message Error message if value is "null"
+ * @throws {TypeError} Throws an error if the value is "null"
+ */
+function ensureValue(val, message) {
+    if (val === null || typeof val === "undefined" || val === types.unset)
+        throw new TypeError(message);
+}
+
+/**
+ * Wraps each of the elements into given subtype
+ * @param {Array} values
+ * @param {rust.ComplexType} subtype
+ * @returns {Array<rust.QueryParameterWrapper>}
+ */
+function encodeListLike(values, subtype) {
+    if (!Array.isArray(values))
+        throw new TypeError(
+            `Not a valid list value, expected Array obtained ${util.inspect(values)}`,
+        );
+
+    let res = [];
+    for (let i = 0; i < values.length; i++) {
+        // This requirement is based on the datastax code
+        ensureValue(
+            values[i],
+            "A collection can't contain null or unset values",
+        );
+        res.push(getWrapped(subtype, values[i]));
+    }
+    return res;
+}
+
+/**
+ *
```

Comment:
:question: Why the additional empty line, does it  change anything when generating docs?